src/extension.ts

```diff
@@ -221,12 +223,15 @@ export function activate(context: VSCode.ExtensionContext) {
 
   oldConfig = getSonarLintConfiguration();
   // Create the language client and start the client.
-  let languageClient = new LanguageClient(
+  languageClient = new LanguageClient(
     "sonarlint-vscode",
     "SonarLint Language Server",
     serverOptions,
     clientOptions
   );
+  // TODO for workspace folder support, remove when possible in a later release
+  languageClient.registerProposedFeatures();
+
   let disposable = languageClient.start();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Identifier 'disposable' is never reassigned; use 'const' instead of 'let'. ([typescript:S3353](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3353))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=16&issues=AWMHLCjyaT4gCmfv8i_5&open=AWMHLCjyaT4gCmfv8i_5)

[ISSUE_KEY]: AWMHLCjyaT4gCmfv8i_5